lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -93,20 +95,28 @@ function requestHandler(request, response) {
         }
       }
 
+      if (params.has('gzip')) {
+        gzip = Boolean(params.get('gzip'));
+      }
+
       // redirect url to new url if present
       if (params.has('redirect')) {
         return setTimeout(sendRedirect, delay, params.get('redirect'));
       }
     }
 
+    if (gzip) {
+      headers['Content-Encoding'] = 'gzip';
+    }
+
     response.writeHead(statusCode, headers);
 
     // Delay the response
     if (delay > 0) {
       return setTimeout(finishResponse, delay, data);
     }
 
-    finishResponse(data);
+    finishResponse(data, gzip);
```

Comment:
if passing it this way, would need to pass into the `delay` path above as well (to get`delay` and `gzip` to work together).

Could also doing the gzip encoding when doing the header? e.g.
```js
if (gzip) {
  headers['Content-Encoding'] = 'gzip';
  data = zlib.gzipSync(data);
}

response.writeHead(statusCode, headers);

// ...
```

Reply:
Done